testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js

```diff
@@ -1116,6 +1117,49 @@ QUnit.module('list integration', {}, () => {
     });
 });
 
+QUnit.module('Drop Down Popup', {
```

Comment:
here is the same as in ddEditor.tests

Reply:
https://github.com/DevExpress/DevExtreme/pull/22834/commits/f94bb40f05bacbe31caa6768f9e639e701d615da